sensors/triggers/kafka/kafka.go

```diff
@@ -27,6 +27,7 @@ import (
 
 	"github.com/argoproj/argo-events/common"
 	"github.com/argoproj/argo-events/common/logging"
+	eskafka "github.com/argoproj/argo-events/eventsources/sources/kafka"
```

Comment:
It seems weird to import a pkg from sources, could you refactor a bit to move the `scram_client.go` file to `common`?

Reply:
I have moved `scram_client.go` to common. I have also renamed symbols SHA{256,512} to SHA{256,512}New to avoid confusion - one might expect `common.SHA512` to be a structure containing some hash metainformation, `common.SHA512New` represents a function which constructs a new `hash.Hash`.
Please tell me if you have thought of better names for them.